index.Rmd

```diff
@@ -42,15 +42,15 @@ Today's tutorial is about constructing a simple spatio-temporal analysis using r
 
 ![R system architecture graph](figs/geoscripting-system-overview.svg){width=100%}
 
-The overall system architecture is comprised of integrated development environments (IDE), engines, packages, bindings, and libraries. Let's start with the engine, the core program that executes the foundation or crucial tasks of the programming language. The most relevant engines in this course are Python, Google Earth Engine, and R. To interact with the engine, we can either code in the command line directly or we can use an IDE, which provides many tools and features for working with the engine integrated in a single environment. An IDE allows the developer to write code, test it, and debug it all in a single software application. For interacting with the R engine, we can use *Rstudio* or *RKWard* for instance. 
+The overall system architecture is comprised of integrated development environments (IDE), engines, packages, bindings, and libraries. Let's start with the engine, the core program that executes the foundation or crucial tasks of the programming language. The most relevant engines in this course are Python, Google Earth Engine, and R. To interact with the engine, we can either code in the command line directly or we can use an IDE, which provides many tools and features for working with the engine integrated in a single environment. An IDE enables developers to write, test, and debug code within a single software application. For interacting with the R engine, we can use *Rstudio* or *RKWard* for instance. 
```

Comment:
Ah, I see. Then I think it might be useful to move it to the place where most of the architecture and its components are explained which is the package and project management. I think putting it in the summary or just above is a good place!